dask/dataframe/dask_expr/_collection.py

```diff
@@ -416,6 +416,8 @@ def __getitem__(self, other):
             other = list(other)
         elif isinstance(other, list):
             other = other.copy()
+        elif is_scalar(other) and hasattr(other, "item"):
```

Comment:
can we check explicitly for numpy scalars here?

Reply:
Oh right- I was looking for a clean way to check if an object is a numpy scalar (since something like `np.isscalar` will return True for `0` as well. Do you know of a good utility for this?